pages/advanced/eip-7702/overview.mdx

```diff
@@ -0,0 +1,42 @@
+# What is EIP-7702?
+
+EIP-7702 is a step towards account abstraction, enabling EOAs (Externally Owned Accounts) to have both code and
+storage. This enhancement allows EOAs to function as smart contract accounts, unlocking new features such as:
+
+- Transaction batching
+- Gas sponsorship
+- Delegated actions: Granting other addresses limited access to act on behalf of the EOA
+
+## Signing Process
+In its current implementation, EIP-7702 requires the EOA to sign a special hash calculated using the following parameters:
+
+- `chain_id`: The identifier of the blockchain network
+- `address`: The account address to which calls will be delegated
+- `nonce`: Current nonce of the account.
+
+
+Setting Code for the EOA
```

Comment:
Is this a title? #'s are missing

Reply:
Actually its a typo. Fixed it. Thanks for noticing. https://github.com/safe-global/safe-docs/pull/651/commits/1628f64f77fcf5a2b12c40b177e29ff7a0ed1975